includes/object-cache.php

```diff
@@ -1169,6 +1173,111 @@ public function add_multiple( array $data, $group = 'default', $expire = 0 ) {
         return $values;
     }
 
+    /**
+     * Adds multiple values to the cache in one call.
+     *
+     * @param array  $data   Array of keys and values to be added.
+     * @param string $group  Optional. Where the cache contents are grouped.
+     * @param int    $expire Optional. When to expire the cache contents, in seconds.
+     *                       Default 0 (no expiration).
+     * @return bool[] Array of return values, grouped by key. Each value is either
+     *                true on success, or false if cache key and group already exist.
+     */
+    protected function add_multiple_at_once( array $data, $group = 'default', $expire = 0 )
+    {
+        $values = [];
+
+        $group = $this->sanitize_key_part( $group );
+
+        if (! $this->is_ignored_group( $group ) && $this->redis_status() ) {
```

Comment:
This is missing support for ignored groups like `delete_multiple_at_once()` has.

Reply:
Doesn't need to check for `redis_status()` again.

Reply:
> This is missing support for ignored groups like `delete_multiple_at_once()` has.

I guess you mean we should add them to the internal cache like in the `add_or_replace` function not redis.

https://github.com/zaherg/redis-cache/blob/better-implementation-for-add-multiple-and-set-multiple/includes/object-cache.php#L1400-L1404